eng/Versions.props

```diff
@@ -92,7 +92,7 @@
     <Tooling_MicrosoftCodeAnalysisTestingVersion>1.0.1-beta1.21103.2</Tooling_MicrosoftCodeAnalysisTestingVersion>
     <MicrosoftVisualStudioShellPackagesVersion>17.0.0-preview-2-31228-438</MicrosoftVisualStudioShellPackagesVersion>
     <MicrosoftVisualStudioPackagesVersion>17.0.35-gdeb9415fdc</MicrosoftVisualStudioPackagesVersion>
-    <RoslynPackageVersion>3.10.0-2.21179.6</RoslynPackageVersion>
+    <RoslynPackageVersion>4.0.0-3.21317.9</RoslynPackageVersion>
```

Comment:
/cc @genlu Allison got your back! (he mentioned how he was seeing some unexpected assembly loads in another convo)